coconut/control/control.go

```diff
@@ -392,6 +393,16 @@ func stringMapToString(stringMap map[string]string, indent string) string {
 	return strings.Join(accumulator, "\n")
 }
 
+func SetEnvironmentName(cxt context.Context, rpc *coconut.RpcClient, cmd *cobra.Command, args []string, o io.Writer) error {
+	if len(args) == 1 { // Set global default
+		fmt.Fprintln(o, "Operation failed.")
+		return fmt.Errorf("Missing Argument")
+	} else if len(args) == 2 { // Set per-repo default
+		_, _ = rpc.SetEnvironmentName(cxt, &pb.SetEnvironmentNameRequest{Name: args[0], Id: args[1]}, grpc.EmptyCallOption{})
```

Comment:
`SetEnvironmentName` should be allowed to fail, i.e. I advise you capture the error value and report it to the user (why the name couldn't be set), and on the server side you implement the failure modes for this call, returning any error conditions using the gRPC status idiom as you already do.